beacon/engine/gen_blockmetadata.go

```diff
@@ -70,13 +69,12 @@ func (b *BlockMetadata) UnmarshalJSON(input []byte) error {
 		return errors.New("missing required field 'txList' for BlockMetadata")
 	}
 	b.TxList = *dec.TxList
-	if dec.HighestBlockID == nil {
-		return errors.New("missing required field 'highestBlockID' for BlockMetadata")
-	}
-	b.HighestBlockID = dec.HighestBlockID
 	if dec.ExtraData == nil {
 		return errors.New("missing required field 'extraData' for BlockMetadata")
 	}
 	b.ExtraData = dec.ExtraData
+	if dec.BasefeeSharingPctg != nil {
```

Comment:
Check `BasefeeSharingPctg` less than `100`.

Reply:
The smart contract should make sure of that?